.vscode/launch.json

```diff
@@ -1,17 +0,0 @@
-{
```

Comment:
Please do not include .vscode files in the PR

Reply:
Hi @zdmc23 , this .vscode files appear in the changes to indicate their elimination. At this moment this files does not exist in my current branch (sprint-8).